### ISSUE TITLE
non-null field error

### ISSUE THREAD

**Issue by jellegerbrandy** [daostack/subgraph#412]
This is in the `test_env` image, but it is also reproducable by running `npm run migrate` and `npm run deploy` in the current subgraph repository.

The error is this:
```
{
  "errors": [
    {
      "locations": [
        {
          "column": 11,
          "line": 55
        }
      ],
      "message": "Null value resolved for non-null field `genesisProtocolParams`"
    },
    {
      "locations": [
        {
          "line": 87,
          "column": 11
        }
      ],
      "message": "Null value resolved for non-null field `gpQueue`"
    }
  ]
}
```
And the (query is this:
```
    query ProposalState
          {
            proposals {
              ...ProposalFields
              votes {
                id
              }
              stakes {
                id
              }
            }
          }
          fragment ProposalFields on Proposal {
          id
    
          genesisProtocolParams {
            activationTime
            boostedVotePeriodLimit
            daoBountyConst
            limitExponentValue
            minimumDaoBounty
            preBoostedVotePeriodLimit
            proposingRepReward
            queuedVotePeriodLimit
            queuedVoteRequiredPercentage
            quietEndingPeriod
            thresholdConst
            votersReputationLossRatio
          }
          
          gpQueue {
            id
            threshold
            votingMachine
          }
     
        }
```

**Comment by jellegerbrandy** [comment]
I am re-opening this issue, because I think this is still a bug (but feel free to close this if you think it is not). 

I  got this error again when I deployed a dao (using `migration.migrateDAO`) on arc version 32 and then generated the subgraph on the basis of migration.json, using subgraph v36

this creates a corrupt subgraph

This  (imho) should just never happen. The subgraph should not write buggy data to the database that break the Proposal query. 

In this case, either the subbgraph generation script should throw an error ("this version is not supported"), or the DAO should simply not be indexed.

**Comment by jellegerbrandy** [comment]
To reproduce this, clone the https://github.com/daostack/test_env repository, and then do:

```
git checkout 8c32fdc04de2a5eb1df46803e24c54feecf70d4a
./release.sh -d
````
This will start up graph-node on the usual port 8000. Then run this query:

```
{proposal (id:"0x2c320fa0e7a341cd28b8140247422b430c941390567ca3690fffc1e8577d3057") {
  id
  gpQueue {id }
}}
```

**Comment by ben-kaufman** [comment]
@jellegerbrandy I've tried repeating the steps above but the error does not replicate and the data returns correctly. Maybe try to restart/ prune your docker and try again.

**Comment by ben-kaufman** [comment]
Fixed with #437
